### PR TITLE
Dev 12.0 afc 2790

### PR DESCRIPTION
**Pull request by berhr**
Översättning av .PO-fil till svenska

### CODE REVIEW

File: mass_mailing_autoresponder/i18n/sv_SE.po [hunk 1]
```diff
@@ -0,0 +1,736 @@
...
+# This file contains the translation of the following modules:
+#	* mass_mailing_autoresponder
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0-20210922\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-09-27 13:42+0000\n"
+"PO-Revision-Date: 2021-09-27 13:42+0000\n"
+"Last-Translator: <>\n"
```
Comment (nils-vertel) [inline mass_mailing_autoresponder/i18n/sv_SE.po]: Add your name here

File: mass_mailing_autoresponder/i18n/sv_SE.po [hunk 2]
```diff
@@ -0,0 +1,736 @@
...
+#: model_terms:ir.ui.view,arch_db:mass_mailing_autoresponder.view_event_email_audit_tree
+msgid "Autoresponder"
+msgstr "Autoresponder"
+
+#. module: mass_mailing_autoresponder
+#: model:ir.actions.server,name:mass_mailing_autoresponder.event_close_when_end_ir_actions_server
+#: model:ir.cron,cron_name:mass_mailing_autoresponder.event_close_when_end
+#: model:ir.cron,name:mass_mailing_autoresponder.event_close_when_end
+msgid "Autoresponder : Update Autoresponder status to end when it finished."
+msgstr "Autoresponder : Uppdatera autoresponderstatus för att sluta när den är klar."
```
Comment (nils-vertel) [inline mass_mailing_autoresponder/i18n/sv_SE.po]: The english version uses A on autoresponder, consider using capital A on the swedish version as well.

File: mass_mailing_autoresponder/i18n/sv_SE.po [hunk 3]
```diff
@@ -0,0 +1,736 @@
...
+
+#. module: mass_mailing_autoresponder
+#: model:ir.actions.act_window,name:mass_mailing_autoresponder.action_all_email_sent
+#: model:ir.actions.act_window,name:mass_mailing_autoresponder.action_schedule_line_emails
+msgid "Autoresponder Mail Audit"
+msgstr "Autoresponder postrevision"
```
Comment (nils-vertel) [inline mass_mailing_autoresponder/i18n/sv_SE.po]: Capital Case on the english version but not in the swedish version?

File: mass_mailing_autoresponder/i18n/sv_SE.po [hunk 4]
```diff
@@ -0,0 +1,736 @@
...
+
+#. module: mass_mailing_autoresponder
+#: model_terms:ir.ui.view,arch_db:mass_mailing_autoresponder.view_event_form
+msgid "Autoresponder Name"
+msgstr "Autoresponder namn"
```
Comment (nils-vertel) [inline mass_mailing_autoresponder/i18n/sv_SE.po]: Capital Case on the english version but not in the swedish version?

File: mass_mailing_autoresponder/i18n/sv_SE.po [hunk 5]
```diff
@@ -0,0 +1,736 @@
...
+
+#. module: mass_mailing_autoresponder
+#: model:ir.model.fields,field_description:mass_mailing_autoresponder.field_partner_event__name
+msgid "Autoresponder Segment"
+msgstr "Autoresponder segment"
```
Comment (nils-vertel) [inline mass_mailing_autoresponder/i18n/sv_SE.po]: Capital Case on the english version but not in the swedish version?

File: mass_mailing_autoresponder/i18n/sv_SE.po [hunk 6]
```diff
@@ -0,0 +1,736 @@
...
+
+#. module: mass_mailing_autoresponder
+#: model:ir.model.fields,field_description:mass_mailing_autoresponder.field_partner_event_email_schedule__bounced
+msgid "Bounced Email"
+msgstr "Studsade e-post"
+
+#. module: mass_mailing_autoresponder
+#: model:ir.model.fields,field_description:mass_mailing_autoresponder.field_partner_event_email_schedule__bounced_ratio
+msgid "Bounced Ratio"
+msgstr "Studsat förhållande"
```
Comment (nils-vertel) [inline mass_mailing_autoresponder/i18n/sv_SE.po]: Capital Case on the english version but not in the swedish version?

File: mass_mailing_autoresponder/i18n/sv_SE.po [hunk 7]
```diff
@@ -0,0 +1,736 @@
...
+
+#. module: mass_mailing_autoresponder
+#: model:ir.model.fields,field_description:mass_mailing_autoresponder.field_partner_event__message_ids
+msgid "Messages"
+msgstr "Meddelanden"
+
+#. module: mass_mailing_autoresponder
+#: model:ir.model.fields,field_description:mass_mailing_autoresponder.field_partner_event__message_needaction_counter
+msgid "Number of Actions"
+msgstr "Antal åtgärder"
```
Comment (nils-vertel) [inline mass_mailing_autoresponder/i18n/sv_SE.po:520]: Capital Case on the english version but not in the swedish version?

**Review by nils-vertel — CHANGES_REQUESTED** [review]

**Comment by HazeLidman** [comment]
okay, mergat

**Pull request merged**